public/scripts/chat-cemplates.js

```diff
@@ -0,0 +1,76 @@
+// https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest
+async function digestMessage(message) {
+    const msgUint8 = new TextEncoder().encode(message); // encode as (utf-8) Uint8Array
+    const hashBuffer = await window.crypto.subtle.digest('SHA-256', msgUint8); // hash the message
```

Comment:
Thanks for pointing that out. I put the hash into the response from the backend to avoid having to bundle anything. It looks reasonable to me, but if you prefer Webpack style I can do that.

Reply:
Right, one less dependency. Switched to using that.